lib/mapping/MapFormatJson.cpp

```diff
@@ -189,7 +189,7 @@ namespace TriggeredEventsDetail
 			break;
 		}
 
-		return VLC->modh->makeFullIdentifier("", metaclassName, identifier);
+		return CModHandler::makeFullIdentifier("", metaclassName, identifier);

```

Comment:
@IvanSavenko so, you prefer to abandon https://clang.llvm.org/extra/clang-tidy/checks/readability/static-accessed-through-instance.html?
There was a case of already static function of VLC, who we should call it by instance?

Reply:
Yeah, probably.
Not against such usage of static methods, so no need to revert existing code, but also see no point in switching to it.

Reply:
Realistically speaking - I don't think that it would help with VLC removal.

Currently my (very long-term) ideas regarding VLC is to either make it constant during the game (solving main issues with it) or to make it member of base library classes, e.g. IObjectInterface (allowing different VLC's for different objects).
But outright removal sound like a fantasy.

So in terms of VLC removal this is as good as removing a pebble from a mountain. We likely have thousands of instances of VLC access, if not more. Can't say that this is a bad change, but I don't see it as good change either. 

If you wish to keep it - OK, up to you I guess?

Reply:
I wish to keep it, because in this place it makes sense (makeFullIdentifier is more like library function instead of method)